packages/opencensus-core/src/trace/model/tracer.ts

```diff
@@ -158,12 +181,14 @@ export class Tracer implements types.Tracer {
    * @param type The span type.
    * @param parentSpanId The parent span ID.
    */
-  startSpan(name?: string, type?: string, parentSpanId?: string): types.Span {
+  startChildSpan(name?: string, type?: string, parentSpanId?: string):
```

Comment:
I believe `parentSpanId` doesn't need to be here? It's guaranteed to be `this.spanId`.

Reply:
Agree.